datacenterlight/views.py

```diff
@@ -484,9 +487,48 @@ def post(self, request, *args, **kwargs):
             'language': get_language(),
         }
 
-        create_vm_task.delay(vm_template_id, user, specs, template,
-                             stripe_customer_id, billing_address_data,
-                             stripe_subscription_obj.id, card_details_dict)
+        billing_address = BillingAddress(
```

Comment:
possible to move the code into a util function? since both `hosting/views.py` and `datacenterlight/views.py` use the same code, should be easier to maintain.